index.html

```diff
@@ -1307,9 +1268,21 @@
           <dfn>currency</dfn>
         </dt>
         <dd>
-          A string containing a currency identifier. The value of
-          <a>currency</a> can be any string that is valid within the currency
-          system indicated by <a>currencySystem</a>.
+          <p>
+            A string containing a currency identifier. The value of
+            <a>currency</a> can be any string that is valid within the currency
+            system indicated by <a>currencySystem</a>.
+          </p>
+          <p>
+            When using [[!ISO4217]], all <a data-cite=
+            "!ecma-402#sec-iswellformedcurrencycode">well-formed</a> 3-letter
+            currency codes are allowed. Their canonical form is upper case.
+            However, the set of combinations of currency code for which
+            localized currency symbols are available is implementation
+            dependent. Where a localized currency symbol is not available, a
+            user agent SHOULD us the [[!ISO4217]] currency code SHOULD be used
+            for formatting.
```

Comment:
Last sentence is messed up; should be either "a user agent should" or "should be used", not both.